app/experimenter/nimbus-ui/src/lib/visualization/utils.tsx

```diff
@@ -27,3 +27,20 @@ export const getTableDisplayType = (
 
   return displayType;
 };
+
+export const humanDate = (date: string | number): string => {
```

Comment:
heh, i did the same thing here https://github.com/mozilla/experimenter/pull/4215/files#diff-38365926cca23693f8cf3838da07364cd7392d7ca2b1146dbdaa69e6e9af5a9fR9-R14